README.md

```diff
@@ -1,2 +1,188 @@
-# ceresdb-java-client
-Java client for CeresDB
+# CeresDB Java Client
+[中文](./docs/README_CN.md)
+
+## Introduction
+CeresDBxClient is a high-performance  Java client for CeresDB. CeresDB is a high-performance, distributed, schema-less, cloud native time-series database that can handle both time-series and analytics workloads.
+
+## Features
+- With the well designed SPI, the network transport layer is extensible. And we provide the default implementation which uses the gRPC framework.
+- The client provides high-performance async streaming write API
+- The client also collects lots of performance metrics by default. These metrics can be configured to write to local file
+- We can take memory snapshots that contains the status of critical objects. The snapshots can also be configured to write to local file, which helps a lot when we diagnose complex problems.
+
+## Data ingestion process
+
+```
+                   ┌─────────────────────┐  
+                   │   CeresDBxClient    │  
```

Comment:
ceresdbx -> ceresdb

Reply:
Names in codes are still ceresdbx, so we keep the doc be consistent with codes.